### PR TITLE
Hindi Translations

### PR DESCRIPTION
**Pull request by notanerdstudent**
Work in progress

### CODE REVIEW

**Comment by cloudcloudskies** [comment]
gg

**Comment by notanerdstudent** [comment]
Update:- 144/385 lines

**Review by cloudcloudskies — APPROVED** [review]

**Comment by notanerdstudent** [comment]
Update :- 241/388

File: hi.yml [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+      Congratulations! Your private hub, **{{name}}**, has been successfully created.
+      To join, create an invite using `/hub invite create` and share the generated code. Then join using `/hub join`.
+
+      - **Generate invite:** `/hub invite create`
+      - **Go public:** `/hub manage`
+      - **Join hub:** `/hub join`
+      - **Edit hub:** `/hub manage`
+      - **Add moderators:** `/hub moderator add`
+
+      Learn more about hubs in our [guide](https://discord-interchat.github.io/docs). Join the [support server]({{{support_invite}}}) for help.
```
Comment (dev-737) [inline hi.yml]: Great work, @notanerdstudent! I noticed that there are still some small parts that are not translated 😅 
image

**Review by dev-737 — CHANGES_REQUESTED** [review]

File: hi.yml (hunk 1 above)
Comment (notanerdstudent) [reply hi.yml]: no u dont

Comment (cloudcloudskies) [reply hi.yml]: lmfao

**Review by dev-737 — APPROVED** [review]
LGTM. Thanks for contributing and being our first contributor! 🎉

**Pull request merged**